grpcx/server.go

```diff
@@ -56,22 +58,71 @@ func WithRequestResponseLogs(l logx.Logger) grpc.UnaryServerInterceptor {
 
 // WithErrorLogs returns a gRPC interceptor for unary calls that instrument requests
 // with logs for the errors.
-func WithErrorLogs(l logx.Logger) grpc.UnaryServerInterceptor {
+func WithErrorLogs(l logx.Logger, options ...func(*withErrorLogsOptions)) grpc.UnaryServerInterceptor {
+	var logOptions = &withErrorLogsOptions{
+		debugLevelCodes: []codes.Code{},
+		infoLevelCodes:  []codes.Code{codes.Canceled, codes.Unknown, codes.InvalidArgument, codes.DeadlineExceeded, codes.NotFound, codes.AlreadyExists, codes.PermissionDenied, codes.ResourceExhausted, codes.FailedPrecondition, codes.Aborted, codes.OutOfRange, codes.Unimplemented, codes.Internal, codes.Unavailable, codes.DataLoss, codes.Unauthenticated},
+	}
```

Comment:
what if by default all the errors are logged as Info and we add an option to configure errors as Debug and another option to don't log it?

Reply:
+1

Reply:
Ok, I've added WithoutLevelCodes option, where you can exclude the codes you don't want.